<launch>

    <node pkg="iloam" type="iScanRegistration" name="iScanRegistration" output="screen">
        <rosparam file="$(find iloam)/config/iloam.yaml" command="load"/>
    </node>

</launch>